src/cts/src/TechChar.h

```diff
@@ -61,6 +63,23 @@ namespace cts {
 
 using utl::Logger;
 
+struct pairHash
```

Comment:
PairHash

Reply:
Why?   All of CTS codes start with lower case letter.

Reply:
Fixed anyway

Reply:
The pattern we use for struct names starts with upper case letters. The structs starting with lowercase should be fixed at some point.

Doing a quick search in CTS code, I've found only two structs starting with lowercase. The other ones look correct.

Reply:
https://google.github.io/styleguide/cppguide.html#Type_Names